ci/infra/openstack/load-balancer.tf

```diff
@@ -2,9 +2,7 @@ resource "openstack_lb_loadbalancer_v2" "lb" {
   name          = "${var.stack_name}-lb"
   vip_subnet_id = "${openstack_networking_subnet_v2.subnet.id}"
 
-  security_group_ids = [
-    "${openstack_compute_secgroup_v2.secgroup_master_lb.id}",
-  ]
+  security_group_ids = "${var.enable_openstack_port_security == true ? ["${openstack_compute_secgroup_v2.secgroup_master_lb.id}", ] : []}"
```

Comment:
This is wrong and I will fix it when I know we can bump terraform-provider-openstack